src/mumble/MainWindow.cpp

```diff
@@ -1694,6 +1696,8 @@ void MainWindow::on_qaUserLocalIgnore_triggered() {
 	p->setLocalIgnore(ignored);
 	if (! p->qsHash.isEmpty())
 		g.db->setLocalIgnored(p->qsHash, ignored);
+	else
+		msgBox(MainWindow::tr("This change could not be saved permanently and is lost on restart because %1 does not have a certificate.").arg(Log::formatClientUser(p, Log::Target)));
```

Comment:
When the user sees this message it might not be entirely clear, what "this change" should be. It'd be better if the message would contain the name of the performed action as well.